app/components/candidate_interface/rejection_reasons_component.html.erb

```diff
@@ -1,15 +1,10 @@
-<details class="govuk-details" data-module="govuk-details">
-  <summary class="govuk-details__summary">
-    <span class="govuk-details__summary-text">
-      See feedback from your previous application
-    </span>
-  </summary>
-
-  <div class="govuk-details__text govuk-!-padding-bottom-0">
-    <% rejected_application_choices.each do |application_choice| %>
-      <%= render(SummaryCardComponent.new(rows: application_choice_rows(application_choice))) do %>
-        <%= render(SummaryCardHeaderComponent.new(title: application_choice.provider.name)) %>
-      <% end %>
+<%= render GovukComponent::Details.new(
+  summary: 'See feedback from your previous application',
+),
+  classes: 'govuk-!-padding-bottom-0' do %>
```

Comment:
Do the classes need to go inside the brackets here?

Reply:
good spot! 😅